Major rework of ecosystem pages (#3060)
open-telemetry/opentelemetry.io: content/en/ecosystem/vendors.md

aliases: [/vendors]
# Note: the keywords after 'cSpell:ignore' must be on the same line, no wrapping, hence the Prettier ignore directive
# prettier-ignore
---

A non-exhaustive list of organizations, offering solutions that consume
OpenTelemetry natively via via [OTLP](/docs/specs/otlp/), such as observability
backends and observability pipelines.

These solutions may be open source, commercial, or both. Some organizations
provide a [distribution](/docs/concepts/distributions/) (of customized
OpenTelemetry components), that provides additional capabilities or for improved
ease of use.

{{% ecosystem/vendor-table %}}

To have your organization listed, submit a PR with an entry added to the
[vendors list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/vendors.yaml).
The entry must include the following[^grace-period-2024-01-01]:

> - a link to the documentation on how your offering consumes OpenTelemetry
>   natively via [OTLP](http://localhost:1313/docs/specs/otlp/).
> - if applicable, a link to your distribution.
> - if applicable, a link that proves that your offering is open source. An open
>   source distribution does not qualify your offering to be marked "open source".
> - a point of contact (GitHub handle or email address), we can reach out for
>   future inquiries.

Reviewer: ```suggestion
- Link to the documentation that details how your offering consumes OpenTelemetry
  natively via [OTLP](http://localhost:1313/docs/specs/otlp/).
- Link to your distribution, if applicable
- Link that proves that your offering is open source, if applicable. An open
  source distribution does not qualify your offering to be marked "open source".
- GitHub handle or email address as a point of contact so that we can reach out
  in case we have questions
```

> Note, that this list is for organizations that consume OpenTelemetry to offer

Reviewer: ```suggestion
Note that this list is for organizations that consume OpenTelemetry and offer
```

Observability to [end users](https://community.cncf.io/end-user-community/).

> If you adopt OpenTelemetry for Observability as an end user and do not provide
> any kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).
> <br /> If you provide a library, service or app made observable through
> OpenTelemetry, see [Integrations](/ecosystem/integrations).

Reviewer: ```suggestion
If you adopt OpenTelemetry for Observability as an end-user organization, and you do not provide
any kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).

If you provide a library, service, or app that is made observable through
OpenTelemetry, see [Integrations](/ecosystem/integrations).
```

aliases: [/vendors]
# Note: the keywords after 'cSpell:ignore' must be on the same line, no wrapping, hence the Prettier ignore directive
# prettier-ignore
---

> A non-exhaustive list of organizations that provide solutions which consume
> OpenTelemetry natively via [OTLP](/docs/specs/otlp/) e.g. observability backends
> or observability pipelines.

Reviewer: ```suggestion
A non-exhaustive list of organizations, offering solutions that consume OpenTelemetry natively via
via [OTLP](/docs/specs/otlp/), such as observability backends and observability pipelines.
```

> Those offerings may be open source, commercial or both. Some organizations

Reviewer: ```suggestion
These solutions may be open source, commercial, or both. Some organizations
```

> provide a [distribution](/docs/concepts/distributions/) (a customized version of
> OpenTelemetry components), that provide additional capabilities or ease of use.

Reviewer: ```suggestion
provide a [distribution](/docs/concepts/distributions/) (of customized
OpenTelemetry components), that provides additional capabilities or for improved ease of use.
```

{{% ecosystem/vendor-table %}}

To have your organization included,
[add an entry to the list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/vendors.yaml)
and submit a PR.

You will need to include[^existing-entries]:

- a link to the documentation on how your offering consumes OpenTelemetry
  natively via [OTLP](http://localhost:1313/docs/specs/otlp/).
- if applicable, a link to your distribution.
- if applicable, a link that proves that your offering is open source. An open
  source distribution does not qualify your offering to be marked "open source".
- a point of contact (GitHub handle or email address), we can reach out for
  future inquiries.

Note, that this list is for organizations that consume OpenTelemetry to offer
Observability to end users.

If you adopt OpenTelemetry for Observability as an end user and do not provide
any kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).
<br /> If you provide a library, service or app made observable through
OpenTelemetry, see [Integrators](/ecosystem/integrations).

[^existing-entries]:
    Organizations that have been part of this list before July 2023 are exempted

>     from these requirements until January, 1th 2024.

Reviewer: Please see the copyedits suggested for the corresponding part of the Adopters page, and also make them here.
